src/slotmap.js

```diff
@@ -274,4 +274,54 @@ function slotMapOf(token) {
     return map;
 }
 
-module.exports = { slotMapOf, packValues };
+/**
+ * Represents the value in the persistent storage.
```

Comment:
There is some information in the PR description that would be really useful here. In particular **1.** of section "The reason I've used persistent storage here" and section "Why it might be an issue".

We should include that info as a JS doc here.